app/models/ems_folder.rb

```diff
@@ -131,6 +131,8 @@ def folder_path_objs(*args)
     folders
   end
 
+  virtual_attribute :folder_path, :string, :uses => :all_relationships
```

Comment:
sorry to nit:
Could you move this up to the other `virtual_attribute` definitions on line 19?

https://github.com/ManageIQ/manageiq/pull/20207/files#diff-4024dcf71764353249c6a46da6726effL19

thanks

Reply:
Done